_herobanners/2.5.0-released.md

```diff
@@ -0,0 +1,3 @@
+---
+---
+<a href="/blog/opensearch-project-2022-recap-and-whats-next/"><img src="/assets/media/herobanners/herobanner-2.5.0-released.gif"></a>
```

Comment:
@nateynateynate - can you point it to this post instead? https://opensearch.org/blog/opensearch-2-5-is-live/